vignettes/sirage.R

```diff
@@ -1,50 +1,59 @@
 ## Definition of the time-step and output as "time"
-dt <- user(1)
+dt <- user(0.25)
 initial(time) <- 0
 update(time) <- (step + 1) * dt
 
 ## Core equations for transitions between compartments:
+update(S_tot) <- S_tot - sum(n_SI)
```

Comment:
Would `update(S_tot) <- sum(S[])` be better, as this better reflects this being a derived quantity from the underlying compartments

Reply:
I tried this but it computes from the previous compartments, I can't change the order..

update(S[]) <- S[i] - n_SI[i]  # this is correct
update(S_tot) <- sum(S[]) # sums the compartments before the line above

hence calculating the sum for removed instead.

Reply:
Ok, that make sense! Maybe just add a comment to the model code to note why this is necessary